### DISCUSSION TITLE
How can I test my Electron app with cypress?

### DISCUSSION THREAD

**Discussion by zxytest** [cypress-io/cypress#14329] [Questions and Help]
In the FAQ part, i saw someone asked this question, and the answer is :Testing your Electron app will not ‘just work’, as Cypress is designed to test anything that runs in a browser and Electron is a browser + Node.

That being said, we use Cypress to test our own Desktop app’s front end - by stubbing events from Electron. These tests are open source so you can check them out here(https://github.com/cypress-io/cypress/tree/develop/packages/desktop-gui/cypress/integration).
I checked out the code, but i still don't know how to run and test the electron client.Can give more details or some samples are better.

**Comment by Changyuan2021** [comment]
I have the same issue, have you got the solution on how to use the cypress to automate the Electron desktop app?
